railties/test/railties/engine_test.rb

```diff
@@ -267,6 +267,22 @@ class AddFirstNameToUsers < ActiveRecord::Migration::Current
       assert_nothing_raised { BukkitController }
     end
 
+    test "can draw routes in app routes from engines" do
+      @plugin.write "config/routes/testing.rb", <<~RUBY
+        Rails.application.routes.draw do
+          get "/testing", to: "test#action", as: :testing
+        end
+      RUBY
+
+      @plugin.write "config/routes.rb", <<~RUBY
+        Rails.application.routes.draw do
+          draw(:testing)
+        end
+      RUBY
+
+      assert_match(/testing GET/, rails("routes"))
```

Comment:
This works, but in this file you can see a lot of tests are just making a request to the path and asserting the result. I think this will be faster than running a command in the console.

Reply:
Ok, will update!